css-brochure-form/styles.css

```diff
@@ -0,0 +1,52 @@
+.center {
+  margin: 0 auto;
+  width: 40%;
```

Comment:
instead of using a percentage width for the `container` I would recommend a fixed width like `px` or `rem` also, be sure to choose a width which more closely matches the original width of the form in the example.